specification/batch/Azure.Batch/routes.tsp

```diff
@@ -140,8 +23,11 @@ alias Created<T> = {
 #suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use camelCase"
 #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Operation 'Exists' should be defined using a signature from the Azure.Core namespace."
 #suppress "@azure-tools/typespec-azure-core/rpc-operation-request-body" "BatchOperation with '@get' cannot have a body."
-@clientName("Batch")
-interface BatchApi {
+@client({
+  name: "BatchClient",
+  service: Azure.Batch,
+})
+interface Batch {
```

Comment:
rename client should be put into the client.tsp, we should never use any configuration from @azure-tools/typespec-client-generator-core within the scope of main.tsp and all the files that it refers to.